relational_operators/HashJoinOperator.cpp

```diff
@@ -20,6 +20,7 @@
 #include "relational_operators/HashJoinOperator.hpp"
 
 #include <algorithm>
+#include <map>
```

Comment:
Could you please point out the location of using `map` in this file? I could not find it.